README.md

```diff
@@ -251,11 +251,10 @@ For a complete example, suppose we want to load data from CSV files with the fol
 - Load csv files
 - Parse csv file and yield rows
 
-To support the above pipeline, `CSVParser` is registered as `parse_csv_files` to consume file streams and expand them as
-rows.
+To support the above pipeline, `CSVParser` is registered as `parse_csv` to consume file streams and expand them as rows.
 
 ```py
-@functional_datapipe("parse_csv_files")
+@functional_datapipe("parse_csv")
```

Comment:
This could break TorchText datasets, can you please verify with them

Reply:
The functional API was changed a long time ago, it is the README file that didn't get updated.

The change here updates the README file, not the any code.

Reply:
I just checked that TorchText has updated their usage as well:

https://github.com/pytorch/text/search?q=parse_csv

Reply:
Cool, just making sure we both fine in terms of releases and will not break someone.